test/unit/specs/components/VHeader/v-filter-button.spec.js

```diff
@@ -83,12 +83,9 @@ describe('VFilterButton', () => {
     mediaQueryListMock.matches = false
     appliedFilters = []
     const { container } = render(VFilterButton, options)
-
-    // Text is only visible for screen readers.
-    const buttonTextLabel = screen.getByText('Filters')
+    // Screen debug shows that there is aria-label, but findByLabelText doesn't
+    // find the button.
```

Comment:
When I tried this locally it appeared to work, but I used `getByLabelText` instead of `find*`. The `find*` methods return a Promise... maybe that was the issue?

Specifically it worked when I did `screen.getByLabelText(/filters/i)`